examples/ui5-js-app/webapp/test/e2e/locators-basic.test.js

```diff
@@ -55,4 +55,26 @@ describe("mixed locators", () => {
         wdi5.getLogger().log("retrievedClassNameStatus", retrievedClassNameStatus)
         expect(retrievedClassNameStatus).toBeTruthy()
     })
+
+    // #291
+    it("should find a sap.m.Select and get an entry", async () => {
+        const selector = {
+            selector: {
+                interaction: "root",
+                controlType: "sap.m.Select",
+                viewName
+            }
+        }
+        /**
+         * @type {import("sap/m/Select").default}
+         */
+        const select = await browser.asControl(selector)
+        await select.open() // <- meh, but needed to have the select items in the DOM
```

Comment:
just out of curiosity. The list has to be opened to receive the `selectedItem`? That is really, meh...

Maybe an alternative approach would be to let wdi5 really "interact" with the control through:
```javascript
browser.asControl({
   "controlType": "sap.m.Select"
    "interaction": {
        "idSuffix": "arrow"
    }
}).press()
which opens the list via an user interaction.
since we only need this step to get the selected item. `open` is also fine for me

Reply:
added it in the test as alternative. thanks, good point!